instruqt-tracks/terraform-basics-aws/track.yml

```diff
@@ -0,0 +1,472 @@
+
+    1. Run the following commands in the terminal to see your temporary credentials:
+
+    ```
+    env | grep AWS_ACCESS_KEY_ID
+    env | grep AWS_SECRET_ACCESS_KEY
+    ```
+  notes:
+  - type: text
+    contents: Did you know HCL stands for "Hashicorp COnfiguration Language"?
+  tabs:
+  - title: Code Editor
+    type: service
+    hostname: workstation
+    port: 8080
+  - title: Shell
+    type: terminal
+    hostname: workstation
+  difficulty: basic
+  timelimit: 300
+- slug: terraform-code
+  id: r2xd5ubxvdnh
+  type: challenge
+  title: "\U0001F468‍\U0001F4BB What does Terraform code look like?"
+  teaser: The terraform DSL (Domain Specific Language) is a declarative language that
+    lets you build almost any type of infrastructure.
+  assignment: |-
+    We've downloaded some terraform code onto your workstation.
+
+    1. Run the following command in the terminal to see the terraform files in your working directory:
+
+    ```
+    ls *.tf
+    ```
+
+    2. Switch tabs to VSCode. You will note the same files are visible in the file explorer pane on the left. Terraform files are marked with the purple T icon.
+
+    Terraform code always ends with a `.tf` extension. You can have as many terraform files as you want, but these three are commonly created first:
+
+    **main.tf** - Where most of your terraform code is stored. This is the part that does the building of resources.
+
+    **variables.tf** - Use this file to define which variables will be available to your users.
+
+    **outputs.tf** - This file contains outputs that will be shown at the end of a successful terraform run.
+
+    Files that end in anything other than `*.tf` or `*.tfvars` are ignored by Terraform.
+  notes:
+  - type: text
+    contents: |-
+      Terraform will read anything in the current directory that ends in `*.tf` or `*.tfvars`.
+
+      By convention most terraform workspaces will contain `main.tf`, `variables.tf`, and `outputs.tf` files.
+
+      You can also group your terraform code into files by purpose. For example, you might place all your load balancer configuration code into a file called `load_balancer.tf`.
+  tabs:
+  - title: Code Editor
+    type: service
+    hostname: workstation
+    port: 8080
+  - title: Shell
+    type: terminal
+    hostname: workstation
+  difficulty: basic
+  timelimit: 300
+- slug: terraform-init-provider
+  id: vb4yy4igazwq
+  type: challenge
+  title: "\U0001F3E1 Terraform Init - Install the Providers"
+  teaser: Terraform needs a provider to talk to cloud APIs. The provider is the bridge
+    that connects terraform core to your infrastructure providers.
+  assignment: |-
+    We have downloaded some terraform code for the HashiCat application. We'll be using this source code for the rest of the track.
+
+    Before we can do anything with terraform we need to initialize our workspace. Run the following    command in your terminal window:
+    ```
+    terraform init
```

Comment:
Again, if the first thing we have the student do is run a command, please have the challenge open directly to the terminal and not the text editor.

Reply:
The behaviour I'm seeing is the behaviour you are describing as the desired state. Let's catch up when you are back from Thanksgiving.